### DISCUSSION TITLE
Facing issue WalletConnect visibility on Android

### DISCUSSION THREAD

**Discussion by ziaakhan7872** [WalletConnect/walletconnect-monorepo#3264] [Developer Support]
Hello, We have successfully published our tomiPAY wallet on WalletConnect. It’s showing up in the WalletConnect list on iOS and working perfectly. However, we’re facing an issue with its visibility on Android. Despite trying various solutions, the problem persists. We need your guidance in this matter. Any help would be highly appreciated.

**Comment by glitch-txs** [comment]
Android works universally with a single deeplink, it should open a mobile modal with all installed wallets as options, different from iOS.

**Reply by ziaakhan7872** [reply]
our wallet is installed and live on the Android store, but still, it's not showing in the mobile modal. can suggest anything which I can implement.

**Reply by glitch-txs** [reply]
Hi, we have this documented here, let me know if helps 
https://docs.walletconnect.com/2.0/android/guides/mobile-linking

**Reply by ziaakhan7872** [reply]
You're team responding too late .it's not a good thing for us. By the way, I found the solution to my problem Thanks

**Reply by glitch-txs** [reply]
Closing as completed, thanks for your feedback 🙏
